webapp/src/components/ActionMenu.tsx

```diff
@@ -55,7 +55,7 @@ export function useShareLink(config: ShareLinkConfig): MenuItem<ShareLinkConfig>
                         isOpen={shareLinkModalOpen}
                         onClose={() => setShareLinkModalOpen(false)}
                         isTester={isTester}
-                        link={config.token ? config.tokenToLink(props.id, config.token) : ""}
```

Comment:
Why do we want to replace all ternary operator(s) with boolean operator(s)?  This fundamentally changes the evaluation logic and can introduce subtle bugs depending on the state of the 2nd operand, especially if that operand is coerced to "falsy"?